docker/prodlike/bitcoin/Dockerfile.bitcoin

```diff
@@ -0,0 +1,8 @@
+FROM debian:bookworm
+
+COPY --from=dobtc/bitcoin:25.1 /opt/bitcoin-*/bin /usr/local/bin
+
+RUN apt-get update && apt-get install -y curl jq zstd
+RUN apt-get clean && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
```

Comment:
Just a little note that this doesn't reduce the final image size as it's still included in a previous layer, unless the image is built with `--squash`.